### PR TITLE
CMake 3.15.0: MSVC Runtime Lib (Static for Py)

### PR DESCRIPTION
**Pull request by ax3l**
Make sure the MSVC runtime libs are static for pybind modules.
https://cmake.org/cmake/help/v3.15/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html

### CODE REVIEW

**Comment by ax3l** [comment]
@robertmaynard would this also be the right thing to add for Python modules?

**Comment by robertmaynard** [comment]
Hmmm 

We need to wait on merging this. Something is cousin has the wrong runtime flags to be set.

**Comment by ax3l** [comment]
> Something is cousin has the wrong runtime flags to be set.

@robertmaynard sorry, the autocorrect on cousin confuses me. Can you rephrase please? :)

**Comment by robertmaynard** [comment]
> > Something is cousin has the wrong runtime flags to be set.
> 
> @robertmaynard sorry, the autocorrect on cousin confuses me. Can you rephrase please? :)

Should have read:

We need to wait on merging this. Something is causing the wrong runtime flags to be set.


Anyway, this response was meant for the AMReX merge request, as I had originally placed the policy update  at the wrong place ( it needs to go before the `project` call) .

**Comment by ax3l** [comment]
I think we don't need this flag here, works well without.
